### ISSUE TITLE
go@1.11.1: hash check failed

### ISSUE THREAD

**Issue by pdepietr** [lukesampson/scoop#2672]

**Comment by r15ch13** [comment]
Please provide more information. The current hash values are correct.
Retry `scoop uninstall go; scoop cache rm go; scoop install go`

**Comment by pdepietr** [comment]
λ scoop uninstall go; scoop cache rm go; scoop install go
ERROR 'go' isn't installed.
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
Installing 'go' (1.11.1) [64bit]
Starting download with aria2 ...
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 1e0889|OK  |    20KiB/s|C:/SDE/Scoop/cache/go#1.11.1#https_storage.googleapis.com_golang_go1.11.1.windows-amd64.zip
Download: Status Legend:
Download: (OK):download completed.
Checking hash of go1.11.1.windows-amd64.zip ... ERROR Hash check failed!
App:         go
URL:         https://storage.googleapis.com/golang/go1.11.1.windows-amd64.zip
First bytes: 3C 21 44 4F 43 54 59 50
Expected:    230a08d4260ded9d769f072512a49bffe8bfaff8323e839c2db7cf7c9c788130
Actual:      e31904679c5766efd1a3ce751addea19c16f09c1277b0685bb0733680de94145

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/lukesampson/scoop/issues/new?title=go%401.11.1%3a+hash+check+failed

**Comment by pdepietr** [comment]
Closing issue. The URL is blocked on my network.
